BUILD_COMMANDLINE.md

```diff
@@ -76,16 +76,33 @@
      ant -lib lib/saxon/saxon-he-11.4.jar build-guidelines-html
      ```
 
-     The results of this build can be found in the web folder (`music-encoding/dist/guidelines/dev/web`). The guidelines are stored in the `index.html` file.
+     The results of this build can be found in the web folder (`music-encoding/dist/guidelines/web`). The guidelines are stored in the `index.html` file.
 
-   * Build a specific customization's RNG schema:
+   * Build the RNG schema of a specific customization:
 
      ```shell
-     ant -lib lib/saxon/saxon-he-11.4.jar -Dcustomization.path="[PATH/TO/YOUR/CUSTOMIZATION]" build-rng
+     ant -lib lib/saxon/saxon-he-11.4.jar -Dcustomization.path="[ABSOLUTE/PATH/TO/YOUR/CUSTOMIZATION]" build-rng
```

Comment:
```suggestion
     ant -lib lib/saxon/saxon-he-11.4.jar -Dcustomization.path="[/ABSOLUTE/PATH/TO/YOUR/CUSTOMIZATION]" build-rng
```

Reply:
(Added a slash at the beginning since it's an absolute path...)

Reply:
Thanks, good catch